test/src/main/java/org/apache/accumulo/test/ScanServerIT_NoServers.java

```diff
@@ -123,16 +127,55 @@ public void testBatchScan() throws Exception {
   }
 
   @Test
-  public void testScanOfflineTable() throws Exception {
-    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+  public void testScanWithNoTserverFallback() throws Exception {
+
+    var clientProps = new Properties();
+    clientProps.putAll(getClientProps());
+    String scanServerSelectorProfiles = "[{'isDefault':true,'maxBusyTimeout':'5m',"
+        + "'busyTimeoutMultiplier':8, 'scanTypeActivations':[], 'timeToWaitForScanServers':'120s',"
+        + "'attemptPlans':[{'servers':'3', 'busyTimeout':'1s'}]}]";
+    clientProps.put("scan.server.selector.impl", ConfigurableScanServerSelector.class.getName());
+    clientProps.put("scan.server.selector.opts.profiles",
+        scanServerSelectorProfiles.replace("'", "\""));
+
+    try (AccumuloClient client = Accumulo.newClient().from(clientProps).build()) {
       String tableName = getUniqueNames(1)[0];
 
       createTableAndIngest(client, tableName, null, 10, 10, "colf");
       client.tableOperations().offline(tableName, true);
 
-      assertThrows(TableOfflineException.class, () -> {
+      assertThrows(TimedOutException.class, () -> {
```

Comment:
A `TableOfflineException` is being thrown instead of a `TimedOutException`

Most likely due to this:
https://github.com/apache/accumulo/blob/3441e148419081266203b32dfcc3a5361cc8baa2/core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java#L800

The test passes successfully if the `client.tableOperations().offline(tableName, true);` operation is removed.